Source/Mosa.Compiler.Framework/Compiler.cs

```diff
@@ -139,7 +139,8 @@ private static List<BaseCompilerStage> GetDefaultCompilerPipeline(CompilerOption
 			return new List<BaseCompilerStage> {
 				new UnitTestStage(),
 				new TypeInitializerStage(),
-				new DevirtualizationStage(),
+
+				//new DevirtualizationStage(),
```

Comment:
Has this stage been disabled intentionally?

Reply:
Yes. However, wait to approve until the next update which will re-enable this.